packages.json

```diff
@@ -16506,5 +16506,21 @@
     "description": "ANOther pure NIm MONGO driver.",
     "license": "MIT",
     "web": "https://mashingan.github.io/anonimongo/src/htmldocs/anonimongo.html"
+  },
+  {
+    "name": "paranim",
+    "url": "https://github.com/paranim/paranim",
+    "method": "git",
+    "tags": [],
```

Comment:
I would recommend you to add some tags, it will be easier for others to search for and find your packages.